addons/binding/org.openhab.binding.chromecast/ESH-INF/thing/thing-types.xml

```diff
@@ -106,10 +236,209 @@
+    </channel-type>
+
+    <channel-type id="duration" advanced="true">
+        <item-type>Number</item-type>
+        <label>Duration</label>
+        <description>Length of currently playing media</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <!-- Metadata Information -->
+    <channel-type id="metadataType" advanced="true">
+        <item-type>String</item-type>
+        <label>Media Type</label>
+        <description>The type of the currently playing media. One of GENERIC, MOVIE, TV_SHOW, AUDIO_TRACK, PHOTO</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="albumArtist" advanced="true">
+        <item-type>String</item-type>
+        <label>Album Artist</label>
+        <description>The name of the album's artist</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="albumName" advanced="true">
+        <item-type>String</item-type>
+        <label>Album Name</label>
+        <description>The name of the album</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="artist" advanced="true">
+        <item-type>String</item-type>
+        <label>Artist</label>
+        <description>The name of the artist</description>
+        <state readOnly="true"/>
     </channel-type>
 
+    <channel-type id="broadcastDate" advanced="true">
+        <item-type>Number</item-type>
+        <label>Broadcast Date</label>
+        <description>The broadcast date of the currently playing media</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="composer" advanced="true">
+        <item-type>String</item-type>
+        <label>Composer</label>
+        <description>The composer of the current track</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="creationDate" advanced="true">
+        <item-type>Number</item-type>
+        <label>Creation Date</label>
+        <description>The creation date of the currently playing media</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="discNumber" advanced="true">
+        <item-type>Number</item-type>
+        <label>Disc Number</label>
+        <description>The disc number of the currently playing media</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="episodeNumber" advanced="true">
+        <item-type>Number</item-type>
+        <label>Episode Number</label>
+        <description>The episode number of the currently playing media</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="height" advanced="true">
+        <item-type>Number</item-type>
+        <label>Height</label>
+        <description>The height of the currently photo</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="images" advanced="true">
```

Comment:
Currently the images channel is only throwing errors for me ("Update channel images: Unsupported value type ArrayList"), but if album cover art is included in that array I think it could be very nice to have.  Perhaps some of the image items in the array could be exposed as their own channels? BTW, very nice work on getting this binding updated!